tests/appsec/test_automated_login_events.py

```diff
@@ -246,11 +246,16 @@ def test_login_wrong_password_failure(self):
                     # Currently in nodejs there is no way to check if the user exists upon authentication failure so
                     # this assertion is disabled for this library.
                     assert meta["appsec.events.users.login.failure.usr.exists"] == "true"
+                    assert meta["appsec.events.users.login.failure.usr.id"] == "social-security-id"
+                else:
+                    assert meta["appsec.events.users.login.failure.usr.id"] == "test"
```

Comment:
not sure why in nodejs here, we dont retrieve the real user id, but the username? @GustavoCaso